src/utils/functionRuntimeUtils.ts

```diff
@@ -42,7 +42,7 @@ export namespace functionRuntimeUtils {
 
                     if (semver.gt(newestVersion, localVersion)) {
                         const canUpdate: boolean = await brewOrNpmInstalled();
-                        const message: string = canUpdate ? localize(
+                        let message: string = canUpdate ? localize(
                             'azFunc.outdatedFunctionRuntimeUpdate',
                             'Your version of the Azure Functions Core Tools ({0}) does not match the latest ({1}). Would you like to update now?',
```

Comment:
Since the button says "Update" you could remove "Would you like to update now?". That might help

Reply:
Here's the shortest message I can think of 
> Please update your Azure Functions Core Tools ({0}) to the latest ({1}).

Reply:
We don't typically use please in our dialogs, do we?  What about
> Update your Azure Functions Core Tools ({0}) to the latest ({1}) for the best experience.

Reply:
Sounds good to me